### PR TITLE
Moving stuff to the public repo.

### PR DESCRIPTION
**Pull request by sebastian-sz**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from setuptools import setup, find_packages
+
+setup(
+    name="tfdata_image_loader",
+    version="0.1",
+    packages=find_packages(),
+    description="A custom module for loading image data for Image Classification problem.",
```
Comment (emilian-godlewski) [inline setup.py:7]: I'd suggest using single quote over double quote - it's more readable and more common. But just a nitpick :)

File: tests/test_image_data_loader.py [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+    data_set = data_loader.load_dataset()  # Batch size 1
+    loaded_images_count = utils_for_tests.calc_dataset_length(data_set)
+
+    assert total_images_count == loaded_images_count
+
+
+def test_loader_correctly_creating_class_names(data_dir, data_loader):
+    class_directories = sorted(os.listdir(data_dir))
+
+    assert (class_directories == data_loader.class_names).all()
```
Comment (emilian-godlewski) [inline tests/test_image_data_loader.py]: What does it do?

File: tests/utils_for_tests.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
+import hashlib
```
Comment (emilian-godlewski) [inline tests/utils_for_tests.py]: I would call just `utils` if it's already in tests folder.

File: tests/utils_for_tests.py [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+        other_content = other_dictionary[key]
+        if isinstance(some_content, np.ndarray):
+            assert (some_content == other_content).all()
+        else:
+            assert some_content == other_content
+
+
+def _dictionaries_keys_equal(some_dictionary: Dict, other_dictionary: Dict):
+    print(sorted(some_dictionary.keys()))
+    print(sorted(other_dictionary.keys()))
```
Comment (emilian-godlewski) [inline tests/utils_for_tests.py]: Why using print?

File: tfdata_image_loader.py [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+        print(self.class_names_mapping)
+        print("")
+
+    def get_image_count(self):
+        return len(list(self.data_dir.glob('*/*')))
+
+    def load_dataset(self):
+        file_names_dataset = self._load_file_names()
+        return (
+            file_names_dataset.map(
```
Comment (emilian-godlewski) [inline tfdata_image_loader.py]: I would use enter before this dot for consistent linebreaking

**Review by anaveld — APPROVED** [review]

**Pull request merged**